pkg_building.es.Rmd

```diff
@@ -1,487 +1,462 @@
 
-Recomendamos encarecidamente a los desarrolladores de paquetes que lean el conciso pero exhaustivo libro de Hadley Wickham y Jenny Bryan sobre el desarrollo de paquetes, que está disponible [gratis en línea](https://r-pkgs.org/). Nuestra guía es parcialmente redundante con otros recursos, pero destaca las directrices de rOpenSci.
+Recomendamos fuertemente a quienes desarrollen paquetes que lean el libro de Hadley Wickham y Jenny Bryan sobre el desarrollo de paquetes. Es conciso pero exhaustivo y está disponible [gratis en línea](https://r-pkgs.org/). Algunas partes de nuetra guía son redundantes con respecto a otros recursos, pero destaca las recomendaciones de rOpenSci.
 
- Para leer por qué merece la pena enviar un paquete a rOpenSci para cumplir las directrices, echa un vistazo a [razones para enviar](#whysubmit). 
+Para leer por qué vale la pena enviar un paquete a rOpenSci siguiendo las recomendaciones, echa un vistazo a [razones para enviar un paquete](#whysubmit). 
 
 ```
 
 ## Nombre del paquete y metadatos
 
-### Cómo nombrar tu paquete
+### Cómo elegir el nombre de tu paquete
 
-- Recomendamos encarecidamente nombres cortos y descriptivos en minúsculas. Si tu paquete tiene que ver con uno o más servicios comerciales, asegúrate de que el nombre no infringe las directrices de la marca. Puedes comprobar si el nombre de tu paquete está disponible, es informativo y no es ofensivo utilizando la función [`available` paquete](https://github.com/ropenscilabs/available). En particular, no *no* elijas un nombre de paquete que ya se utilice en CRAN o en Bioconductor.
+- Recomendamos fuertemente nombres cortos y descriptivos en minúsculas. Si tu paquete tiene que ver con uno o más servicios comerciales, asegúrate de que el nombre no infringe las recomendaciones de la marca. Puedes comprobar si el nombre de tu paquete está disponible, es informativo y no es ofensivo utilizando el [paquete `available`](https://github.com/ropenscilabs/available). En particular, *no* elijas un nombre de paquete que ya se utilice en CRAN o en Bioconductor.
 
-- Un nombre de paquete más único podría ser más fácil de rastrear (para que tú y nosotros evaluemos el uso del paquete) y de buscar (para que los usuarios lo encuentren y busquen en Google sus preguntas). Evidentemente, un nombre *demasiado* nombre de paquete demasiado único puede hacer que el paquete sea menos descubrible (por ejemplo, podría ser un argumento para nombrar tu paquete [geojson](https://github.com/ropensci/geojson)).
+- Un nombre de paquete menos común podría ser más fácil de seguir (para que tú y en rOpenSci evaluemos el uso del paquete) y de buscar (para que quienes lo usen lo encuentren y busquen en Google sus preguntas). Evidentemente, un nombre *demasiado* único puede hacer que el paquete sea más difícil de encontrar (por ejemplo, podría ser un argumento para nombrar tu paquete [geojson](https://github.com/ropensci/geojson)).
```

Comment:
Creo que en vez de "seguir", "rastrear" es más claro. Interpreto que un nombre único-poco común hace fácil rastrear/buscar donde se usa el paquete. 

Sobre lo segundo, la primera vez me pareció razonable. Ahora no le encuentro sentido, ni siquiera a la versión en inglés: Obviously a too unique package name might make the package less discoverable (e.g. it might be an argument for naming your package [geojson](https://github.com/ropensci/geojson)).

Reply:
Tal vez el ejemplo no tiene tanto sentido para nosotras.  Entiendo que está hablando de justificar porqué seleccionaste un nombre taaaan general. Algo como, le pongo geojson porque todo el mundo va a buscarlo asi. Tal vez es necesario hacer eso más explicito en el texto en ambos idiomas y no dejarlo a que el lector sepa que geojson es un nombre no común o muy general (si mi interpretación es correcta)

Reply:
Me gusta tu sugerencia. Solo me preocupa cambiar la estructura del párrafo (oraciones, paréntesis) porque no se como se lleva eso con el futuro sistema que actualice las traducciones. Esta es la razón principal por la cuál hemos sido bastante conservadores hasta ahora.

Reply:
Tenés razón.  Interesante problema. Entonces algo como  _Un nombre de paquete menos común podría ser más fácil de encontrar (para que tú y en rOpenSci evaluemos el uso del paquete) y de buscar (para que quienes lo usen lo encuentren y busquen en Google sus preguntas)_

Reply:
```suggestion
- Un nombre de paquete menos común podría ser más fácil de encontrar (para que tú y en rOpenSci evaluemos el uso del paquete) y de buscar (para que quienes lo usen lo encuentren y busquen en Google sus preguntas). Evidentemente, un nombre *demasiado* único puede hacer que el paquete sea más difícil de encontrar (por ejemplo, podría ser un argumento para nombrar tu paquete [geojson](https://github.com/ropensci/geojson)).
```